### PR TITLE
Clarify harbor support statement

### PR DESCRIPTION
**Pull request by laidbackware**
The Harbor statements in the vSphere docs need to be clarified to make clear that Helm Harbor and VM based Harbor deployments are supported by GSS only when used to host bootstrap images in air-gapped or internet restricted deployments.

### CODE REVIEW

**Review by craigtracey — APPROVED** [review]
This looks great to me. Thanks @laidbackware!

File: src/reference-designs/tko-on-vsphere-nsx.md [hunk 1]
```diff
@@ -320,11 +320,11 @@ We propose deploying Tanzu Kubernetes Grid on non-stretched vSphere clusters. Al
...
-- [**Helm based deployment**](https://goharbor.io/docs/2.1.0/install-config/harbor-ha-helm/) to a Kubernetes cluster - this installation method may be preferred for customers already invested in Helm.
-- [**VM-based deployment]**(https://goharbor.io/docs/2.1.0/install-config/installation-prereqs/) using docker-compose - this installation method is recommended in cases where Tanzu Kubernetes Grid is installed in an air-gapped environment and no pre-existing Kubernetes clusters exist on which to install Harbor. When Kubernetes is already available, the Helm based deployment also works for air-gapped environments.
+- [**Helm based deployment**](https://goharbor.io/docs/2.1.0/install-config/harbor-ha-helm/) to a Kubernetes cluster - this installation method may be preferred for customers already invested in Helm. Helm deployments are only supported by the Open Source community and not by VMware Global Support Services.
```
Comment (srinivasme21) [inline src/reference-designs/tko-on-vsphere-nsx.md:326]: Helm-based deployment

File: src/reference-designs/tko-on-vsphere-nsx.md [hunk 2]
```diff
@@ -320,11 +320,11 @@ We propose deploying Tanzu Kubernetes Grid on non-stretched vSphere clusters. Al
...
+- [**VM-based deployment]**(https://goharbor.io/docs/2.1.0/install-config/installation-prereqs/) using docker-compose - this installation method is recommended in cases where Tanzu Kubernetes Grid is installed in an air-gapped environment and no pre-existing image registry exists to host the TKG system images. VM-based deployments are only supported by VMware Global Support Services to host the system images for air-gapped or internet-less deployments and must not be used for hosting application images.
```
Comment (srinivasme21) [inline src/reference-designs/tko-on-vsphere-nsx.md:327]: ... exists to host the Tanzu Kubernetes Grid system images. In an air-gapped or Internet-less deployment, VMware Global Support Services only support hosting system images on VM-based deployments. Hosting application images is not supported.

File: src/reference-designs/tko-on-vsphere-with-tanzu.md [hunk 3]
```diff
@@ -310,10 +310,13 @@ The initial configuration and setup of the platform does not require any externa
 
 When vSphere with Tanzu is deployed on VDS networking, you can deploy an external container registry (Harbor) for Tanzu Kubernetes clusters.
 
-There are two main supported installation methods for Harbor:
+There are three officially documented installation methods for Harbor:
```
Comment (srinivasme21) [inline src/reference-designs/tko-on-vsphere-with-tanzu.md:313]: The following installation methods are supported for Harbor:

File: src/reference-designs/tko-on-vsphere-with-tanzu.md [hunk 4]
```diff
@@ -310,10 +310,13 @@ The initial configuration and setup of the platform does not require any externa
...
 
-*   [Helm-based deployment](https://goharbor.io/docs/2.1.0/install-config/harbor-ha-helm/) to a Kubernetes cluster: This installation method may be preferred if you are already invested in Helm.
-*   [VM-based deployment](https://goharbor.io/docs/2.1.0/install-config/installation-prereqs) using `docker-compose`: This installation method is recommended in cases where Tanzu Kubernetes Grid is installed in an air-gapped environment and no pre-existing Kubernetes clusters exist on which to install Harbor.
+
+*   [**Tanzu Kubernetes Grid Package deployment**](https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/1.4/vmware-tanzu-kubernetes-grid-14/GUID-packages-harbor-registry.html) on a Tanzu Kubernetes Grid shared services cluster cluster - Tanzu Kubernetes Grid includes signed binaries for Harbor, which you can deploy into a shared services cluster to provide container registry services for other Tanzu Kubernetes (workload) clusters. This installation method is recommended for general use cases.
+air-gapped or internet-less deployments and must not be used for hosting application images.
```
Comment (srinivasme21) [inline src/reference-designs/tko-on-vsphere-with-tanzu.md:317]: *Note*: Do host application images in an air-gapped or Internet-less environment.

File: src/reference-designs/tko-on-vsphere-nsx.md [hunk 5]
```diff
@@ -320,11 +320,11 @@ We propose deploying Tanzu Kubernetes Grid on non-stretched vSphere clusters. Al
 
 ## Container Registry
 
-The Tanzu Editions include [Harbor](https://goharbor.io/) as a container registry. Harbor provides a location for pushing, pulling, storing, and scanning container images used in your Kubernetes clusters. There are three main supported installation methods for Harbor:
+The Tanzu Editions include [Harbor](https://goharbor.io/) as a container registry. Harbor provides a location for pushing, pulling, storing, and scanning container images used in your Kubernetes clusters. There are three officially documented installation methods for Harbor:
```
Comment (srinivasme21) [inline src/reference-designs/tko-on-vsphere-nsx.md:323]: You can install Harbor using the following methods:

File: src/reference-designs/tko-on-vsphere-with-tanzu.md [hunk 6]
```diff
@@ -310,10 +310,13 @@ The initial configuration and setup of the platform does not require any externa
...
+*   [Helm-based deployment](https://goharbor.io/docs/2.1.0/install-config/harbor-ha-helm/) to a Kubernetes cluster: This installation method may be preferred if you are already invested in Helm. Helm deployments are only supported by the Open Source community and not by VMware Global Support Services.
+*   [VM-based deployment](https://goharbor.io/docs/2.1.0/install-config/installation-prereqs) using `docker-compose`: This installation method is recommended in cases where Tanzu Kubernetes Grid is installed in an air-gapped environment and no pre-existing Kubernetes clusters exist on which to install Harbor. VM-based deployments are only supported by VMware Global Support Services to host the system images for
```
Comment (srinivasme21) [inline src/reference-designs/tko-on-vsphere-with-tanzu.md:319]: Replace the last sentence with the following:
In an air-gapped or Internet-less deployment, VMware Global Support Services only supports hosting system images on VM-based deployments. Hosting application images is not supported.

File: src/reference-designs/tko-on-vsphere.md [hunk 7]
```diff
@@ -446,13 +446,13 @@ VMware recommends that you deploy Tanzu Kubernetes Grid on non-stretched vSphere
 
 ## Container Registry
 
-The Tanzu Editions include [Harbor](https://goharbor.io/) as a container registry. Harbor provides a location for pushing, pulling, storing, and scanning container images to be used in your Kubernetes clusters. There are three main supported installation methods for Harbor:
+The Tanzu Editions include [Harbor](https://goharbor.io/) as a container registry. Harbor provides a location for pushing, pulling, storing, and scanning container images to be used in your Kubernetes clusters. There are three officially documented installation methods for Harbor:
```
Comment (srinivasme21) [inline src/reference-designs/tko-on-vsphere.md:449]: You can install Harbor using one of the following methods:

File: src/reference-designs/tko-on-vsphere.md [hunk 8]
```diff
@@ -446,13 +446,13 @@ VMware recommends that you deploy Tanzu Kubernetes Grid on non-stretched vSphere
...
 
-* **[VM-based deployment](https://goharbor.io/docs/2.1.0/install-config/installation-prereqs/)** using docker-compose - VMware recommends using this installation method in cases where Tanzu Kubernetes Grid is being installed in an air-gapped or Internet-less environment and no pre-existing Kubernetes clusters exist on which to install Harbor. When Kubernetes is already available, the Helm based deployment also works for air-gapped environments.
+* **[VM-based deployment](https://goharbor.io/docs/2.1.0/install-config/installation-prereqs/)** using docker-compose - VMware recommends using this installation method in cases where Tanzu Kubernetes Grid is being installed in an air-gapped or Internet-less environment and no pre-existing image registry exists to host the TKG system images. When Kubernetes is already available, the Helm based deployment also works for air-gapped environments. VM-based deployments are only supported by VMware Global Support Services to host the system images for air-gapped or internet-less deployments and must not be used for hosting application images.
```
Comment (srinivasme21) [inline src/reference-designs/tko-on-vsphere.md:455]: ...`docker-compose`... exists to host the Tanzu Kubernetes Grid system images. In an air-gapped or Internet-less deployment, VMware Global Support Services only supports hosting system images on VM-based deployments. Hosting application images is not supported.

File: src/reference-designs/tko-on-vsphere-with-tanzu.md (hunk 4 above)
Comment (benjvi) [reply src/reference-designs/tko-on-vsphere-with-tanzu.md:317]: @laidbackware I think this line was what lead to my confusion

**Comment by craigtracey** [comment]
@laidbackware I am keeping this in a holding pattern as you clarify support levels on your end. Please update if you have any additional thoughts.

File: src/reference-designs/tko-on-vsphere-nsx.md (hunk 1 above)
Comment (laidbackware) [reply src/reference-designs/tko-on-vsphere-nsx.md]: This should be addressed in the latest push

File: src/reference-designs/tko-on-vsphere-nsx.md (hunk 2 above)
Comment (laidbackware) [reply src/reference-designs/tko-on-vsphere-nsx.md]: This should be addressed in the latest push

File: src/reference-designs/tko-on-vsphere-with-tanzu.md (hunk 3 above)
Comment (laidbackware) [reply src/reference-designs/tko-on-vsphere-with-tanzu.md]: This should be addressed in the latest push

File: src/reference-designs/tko-on-vsphere-with-tanzu.md (hunk 4 above)
Comment (laidbackware) [reply src/reference-designs/tko-on-vsphere-with-tanzu.md]: This should be addressed in the latest push

File: src/reference-designs/tko-on-vsphere-nsx.md (hunk 5 above)
Comment (laidbackware) [reply src/reference-designs/tko-on-vsphere-nsx.md]: This should be addressed in the latest push

File: src/reference-designs/tko-on-vsphere-with-tanzu.md (hunk 6 above)
Comment (laidbackware) [reply src/reference-designs/tko-on-vsphere-with-tanzu.md]: This should be addressed in the latest push

File: src/reference-designs/tko-on-vsphere.md (hunk 7 above)
Comment (laidbackware) [reply src/reference-designs/tko-on-vsphere.md]: This should be addressed in the latest push

File: src/reference-designs/tko-on-vsphere.md (hunk 8 above)
Comment (laidbackware) [reply src/reference-designs/tko-on-vsphere.md]: This should be addressed in the latest push

**Pull request merged**